README.md

```diff
@@ -130,6 +131,21 @@ We use jenkins to deploy. Make sure you can access [https://jenkins.eanadev.org/
 
 The test and acceptance jobs should be run off the `develop` branch.
 
+## Docker (Metis UI)
+
+To make a (parameterised) docker image of the app first run these commands from the root of the project:
+
+`npm run build-shared`
+`npm run dist-metis`
+
+and then send the output of that command to a docker nginx image with this command:
+
+`docker build -t metis-ui-app-image:version projects/metis/`
+
+The generated docker image can then be ran with the command:
```

Comment:
add that the values of the variables in the `env_file` needed to be in place before building the image.

Reply:
Changed.

Reply:
yes. as a new user took me a while to figure this out :joy: that is why I suggested it.